src/lib/httpClient.ts

```diff
@@ -64,6 +64,22 @@ export default class httpClient {
         }, log)
     }
 
+    getScreenshotData(buildId: string, baseline: boolean, log: Logger) {
+        return this.request({
+            url: '/getScreenshotData',
```

Comment:
Changes the route to `/screenshot`

Reply:
We already have a route with /screenshot in LSRS, tough its a POST method.